### DISCUSSION TITLE
Zoom level setting SEB

### DISCUSSION THREAD

**Discussion by venom-marvel** [SafeExamBrowser/seb-win-refactoring#174] [Q&A]
Next question on seb is :
My page does not fit into seb. I need to reduce the zoom scale. On wich setting i can do it?

**Comment by dbuechel** [comment]
You can enable the zoom functionality in the configuration tool, see https://safeexambrowser.org/windows/win_usermanual_en.html#AppearancePane. There however isn't a default zoom level that you could set in a configuration. What exactly do you mean with "my page does not fit into seb"? Could you share an image?

**Reply by venom-marvel** [reply]
I mean the default zoom level. So that it fit in the screen corners

**Reply by danschlet** [reply]
I think your web page is bad designed. Show us a screen shot in a standard Chrome browser and the same page in SEB.

**Reply by dbuechel** [reply]
The default zoom level is 100% and can't be changed, at least at the moment.
